test/uniq.js

```diff
@@ -0,0 +1,59 @@
+var shell = require('..');
+
+var assert = require('assert'),
+    fs = require('fs');
+
+shell.config.silent = true;
+
+shell.rm('-rf', 'tmp');
+shell.mkdir('tmp');
+
+var result;
+
+result = shell.uniq();
+assert.ok(shell.error());
+assert.ok(result.code);
+
+assert.equal(fs.existsSync('/asdfasdf'), false); // sanity check
+result = shell.sort('/adsfasdf'); // file does not exist
+assert.ok(shell.error());
+assert.ok(result.code);
+
+//uniq file1
```

Comment:
Please mark this section with:

```
//
// Valids
//
```

See [here](https://github.com/shelljs/shelljs/blob/master/test/cat.js#L26-28) as an example.